README.md

```diff
@@ -82,6 +82,9 @@ yarn
 
 ### Building packages
 
+Make sure you have installed `solc` compiler binary on your machine
```

Comment:
Shouldn't this state that it is optional?

Reply:
yes, I should change this to optional